src/index.mjs

```diff
@@ -65,16 +65,28 @@ export function listen(options) {
 
   const timeoutFn = options.timeoutFn || requestIdleCallback;
 
+  // ray test touch <
+  const prefetchChunks = options.prefetchChunks;
```

Comment:
We are reading `prefetchChunks` from the `options`,  but have not written JSDoc comments for it in the comments higher up (the same way other options are documented). This should probably be changed.

Reply:
So we also should document prefetchChunks option in README as other options are documented like [here](https://github.com/GoogleChromeLabs/quicklink#api).

Reply:
Done